apis/fluentbit/v1alpha2/clusterfluentbitconfig_types.go

```diff
@@ -80,6 +101,8 @@ type Service struct {
 	LogLevel string `json:"logLevel,omitempty"`
 	// Optional 'parsers' config file (can be multiple)
 	ParsersFile string `json:"parsersFile,omitempty"`
+	// Configure a global environment for the storage layer in Service
```

Comment:
@karan56625 I think we should add a note here to tell users it's better to set up volume and volumemount separately for this storage, it should be hostpath type for fluentbit daemonset. we shouldn't write chuncks directly to containers. And the volume & volumemount can be added like https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L129

We'd better add file buffer config to https://github.com/fluent/fluent-operator/blob/master/charts/fluent-operator/values.yaml#L129 too

Reply:
@benjaminhuo ,  I have added a note in that comment. Also one example is put in the values.yaml. Let me know if any change is required in those.